bus/event_bus.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bus
+
+import asaskevichEventBus "github.com/asaskevich/EventBus"
```

Comment:
It's better to have some generic name. If we will decide to replace `asaskevich/EventBus` by another package it will be easy to replace only in one place. 
```suggestion
import eventBus "github.com/asaskevich/EventBus"
```

Reply:
The idea is that we are using `asaskevich` implementation either way, and it has its own interface. `simpliefiedEventBus` is basically an adapter between `EventBus` (our interface) and `asaskevich/EventBus.BusEventBus`. If we want to use another library, than that other library will have different interface, and this adapter wont work - we will have to modify it. That's why indicating clearly that this is just an adapter makes sense to me.